README.md

```diff
@@ -1,32 +1,100 @@
-Example usage:
+# stackhpc.beegfs
+
+This role was last tested on Ansible version 2.5.2.
+
+The role requires `/etc/hosts` file on each node to contain all nodes
+named in the inventory. This means running the following Ansible task:
+
+    ---
+    - name: Generate /etc/hosts file
+      template:
+        src: hosts.j2
+        dest: /etc/hosts
+      become: yes
+    ...
+
+Where `hosts.j2` is a jinja2 template containing:
+
+    # {{ ansible_managed }}
+    127.0.0.1   localhost localhost.localdomain localhost4
+    localhost4.localdomain4
+    ::1         localhost localhost.localdomain localhost6
+    localhost6.localdomain6
+
+    {% for item in ansible_play_hosts %}
+    {% set short_name = item.split('.') %}
+    {{ hostvars[item]['ansible_host'] }}  {{ item }} {{ short_name[0] }}
```

Comment:
whatever the ilab equivalent is?

Reply:
But more generally, in BeeGFS terms? Are the relevant IPs referenced anywhere in the BeeGFS config?

Reply:
Ok, so do the hostnames in /etc/hosts need to map to the IP address used by the nodes when communicating with the configured management IP?

Reply:
yup

Reply:
This one still needs fixing - ansible_host might not be an IP. I think something like this would be sufficient here:

> Each node's hostname must resolve to the IP address used to reach the management host, as configured via `beegfs_host_mgmt`. This may be done via DNS or `/etc/hosts`.